Automattic/wp-calypso
# Untangling: Add Advanced Tools -> Database

Fixes:

- https://github.com/Automattic/dotcom-forge/issues/9620

## Proposed Changes

This PR adds the Advanced Tools -> Database subtab under the `untangling/hosting-menu` flag.

<img width="847" alt="image" src="https://github.com/user-attachments/assets/2a0c23e5-807b-4a90-ba6e-8a6468eb255b">

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites
2. Click an Atomic site
3. Go to Advanced Tools -> Database subtab.
4. Verify that this tool is working.
5. You can also test on a site which does not allow "advanced hosting features" such as SFTP and Database by creating an Ecommerce trial site from https://wordpress.com/ecommerce/:
    - Verify that you can't see this feature both in Advanced Tools and in the original Server Settings.
7. Regression testing:
    - Go to Server Settings
    - Verify that the original `Database access` card is still working.

## Pre-merge Checklist
